frontend/talentsearch/src/js/components/profile/ProfilePage/ProfilePage.stories.tsx

```diff
@@ -2,74 +2,51 @@ import React from "react";
 import { Meta, Story } from "@storybook/react";
 import { fakeUsers } from "@common/fakeData";
 import fakeExperiences from "@common/fakeData/fakeExperiences";
+import { JobLookingStatus } from "@common/api/generated";
 import { ProfilePage, ProfileForm } from "./ProfilePage";
 import { User } from "../../../api/generated";
 
-const fakeUserArray = fakeUsers(5);
+const fakeUserData = fakeUsers(1)[0];
+const fakeExperienceArray = fakeExperiences(3);
 
 export default {
   component: ProfilePage,
   title: "Profile Form",
   args: {},
+  parameters: {
+    apiResponses: {
+      getMyStatus: {
+        data: {
+          me: {
+            isProfileComplete: true,
+            jobLookingStatus: JobLookingStatus.OpenToOpportunities,
+          },
+        },
+      },
+    },
+  },
 } as Meta;
 
-const TemplateProfilePage: Story<User> = (args) => {
+const Template: Story<User> = (args) => {
   return <ProfileForm profileDataInput={args} />;
 };
 
-export const ProfilePageStory1 = TemplateProfilePage.bind({});
-export const ProfilePageStory2 = TemplateProfilePage.bind({});
-export const ProfilePageStory3 = TemplateProfilePage.bind({});
-export const ProfilePageStory4 = TemplateProfilePage.bind({});
-export const ProfilePageStory5 = TemplateProfilePage.bind({});
-export const ProfilePageNull = TemplateProfilePage.bind({});
+export const CompletedWithoutExperiences = Template.bind({});
+export const CompletedWithExperiences = Template.bind({});
+export const EmptyAllNull = Template.bind({});
 
-ProfilePageStory1.args = { ...fakeUserArray[0] };
-ProfilePageStory2.args = { ...fakeUserArray[1] };
-ProfilePageStory3.args = {
-  ...fakeUserArray[2],
-  experiences: fakeExperiences(3),
+CompletedWithoutExperiences.args = { ...fakeUserData };
+CompletedWithExperiences.args = {
+  ...fakeUserData,
+  experiences: fakeExperienceArray,
 };
-ProfilePageStory4.args = {
-  ...fakeUserArray[3],
-  experiences: fakeExperiences(4),
-};
-ProfilePageStory5.args = {
-  ...fakeUserArray[4],
-  experiences: fakeExperiences(5),
-};
-ProfilePageNull.args = {
-  firstName: null,
-  lastName: null,
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const nullUserData: any = {};
+Object.keys(fakeUserData).forEach((key) => {
+  nullUserData[key] = null;
+});
```

Comment:
this being an override doesn't seem to be a big deal to me, but I am just curious, for learning sake is there an elegant way to not need this?
I understand why you did it this way, I am just personally curious if there is a typescript workaround for these cases